tests/bttvIntegration.test.js

```diff
@@ -0,0 +1,166 @@
+const m = require('../src/scripts/bttvIntegration.js');
```

Comment:
Would you be open to destructuring our inputs here?

Reply:
Absolutely! Will do that now